pkg/trace/stats/aggregation.go

```diff
@@ -65,8 +68,13 @@ func getStatusCode(s *pb.Span) uint32 {
 	return uint32(c)
 }
 
+func spanKindIsClientOrProducer(spanKind string) bool {
```

Comment:
```suggestion
func clientOrProducer(spanKind string) bool {
```

nit: just a personal preference to shorten this a bit